parser/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
+1 
@elaksana I'm not sure why the codebase for the library has been included here...